components/schemes/Color/tests/unit/MDCSemanticColorSchemeTests.m

```diff
@@ -177,4 +177,23 @@ - (BOOL)compareColorsWithFloatPrecisionFirstColor:(UIColor *)firstColor
           MDCCGFloatEqual(fAlpha, sAlpha));
 }
 
+- (void)testColorSchemeCopyTest {
+  // Given
+  MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] init];
+  MDCSemanticColorScheme *colorSchemeCopy = [colorScheme copy];
+
+  // Then
+  XCTAssertFalse(colorScheme == colorSchemeCopy);
```

Comment:
Please use `XCTAssertEqual` to make the generated failure message more descriptive.

Reply:
Done.